doc/docs/security/selinux.md

```diff
@@ -0,0 +1,25 @@
+<!-- markdownlint-disable-file MD013-->
+# BlueChi's SELinux policy
+
+BlueChi provides a custom SELinux policy, limiting access of the `bluechi-controller` and `bluechi-agent`. It can be installed via
+
+```bash
+dnf install bluechi-selinux
+```
+
+By default, the package adds an allow rule for the `bluechi-agent` to connect to BlueChi's default port 842. If the controller port where the agent is connecting to changes, then an AVC entry is created:
+
+```bash
+time->Thu Mar 28 14:12:35 2024
+type=PROCTITLE msg=audit(1711635155.532:22): proctitle="/usr/libexec/bluechi-agent"
+type=SYSCALL msg=audit(1711635155.532:22): arch=c000003e syscall=42 success=no exit=-115 a0=a a1=555f97c27c50 a2=10 a3=7ffd4b114d74 items=0 ppid=1 pid=531 auid=4294967295 uid=0 gid=0 euid=0 suid=0 fsuid=0 egid=0 sgid=0 fsgid=0 tty=(none) ses=4294967295 comm="bluechi-agent" exe="/usr/libexec/bluechi-agent" subj=system_u:system_r:bluechi_agent_t:s0 key=(null)
+type=AVC msg=audit(1711635155.532:22): avc:  denied  { name_connect } for  pid=531 comm="bluechi-agent" dest=8450 scontext=system_u:system_r:bluechi_agent_t:s0 tcontext=system_u:object_r:unreserved_port_t:s0 tclass=tcp_socket permissive=1
+```
+
+This rule, however, is not enforcing, but permissive and thus not preventing the `bluechi-agent` to connect to the different host.
```

Comment:
I'd rephrase:

"However this rule is not enforcing, but permissive, and thus not preventing the `bluechi-agent` to connect to the different host."

Reply:
Changed.